src/App.jsx

```diff
@@ -11,37 +11,35 @@ export const App = () => {
   const DONE = "完了";
   const TODO_STATUS = [NOT_START, DONE];
 
-  const todos = [
-    {
-      id: useId(),
-      todoName: 'AAA',
-      todoLimit: '2023-08-22',
-      todoStatus: NOT_START,
-    },
-    {
-      id: useId(),
-      todoName: 'BBB',
-      todoLimit: '2023-08-22',
-      todoStatus: NOT_START,
-    },
-    {
-      id: useId(),
-      todoName: 'CCC',
-      todoLimit: '2023-08-22',
-      todoStatus: NOT_START,
-    },
-  ];
+  // ランダムのIDを生成
+  const getKey = () => Math.random().toString(32).substring(2, 5);
+
+  // タスク情報のState
+  const [todoItems, setTodoItems] = useState([]);
+
+  // TodoFormの入力値をsetTodoItems()にセット
+  const handleTodoAdd = (todoText, todoLimit) => {
+    setTodoItems([
+      ...todoItems,
+      {
+        id: getKey(),
+        todoText: todoText,
+        todoLimit: todoLimit,
+        todoStatus: NOT_START,
+      }
+    ]);
```

Comment:
その書き方でも良いんですが、こう書くほうが良いという意見もあります。
動作は問題ないので一旦はどちらでも構いません

```suggestion
    setTodoItems((prev) => [
      ...prev,
      {
        id: getKey(),
        todoText: todoText,
        todoLimit: todoLimit,
        todoStatus: NOT_START,
      },
    ]);
```